terraform/meta/bootstrap-env/environment.tf

```diff
@@ -4,6 +4,7 @@ locals {
 
 # Environments often need to know the base org that everything is in
 resource "local_file" "cf_org" {
+  count           = (var.populate_creds_locally == true) ? 1 : 0
```

Comment:
Defaults to `false`. `populate_creds_locally` will be set true only for local env `terraform apply -var populate_creds_locally=true`